forge-gui/res/cardsfolder/upcoming/untimely_malfunction.txt

```diff
@@ -0,0 +1,8 @@
+Name:Untimely Malfunction
+ManaCost:1 R
+Types:Instant
+A:SP$ Charm | Choices$ DBDestroy,DBChangeTargets,DBCantBlock
+SVar:DBDestroy:DB$ Destroy | ValidTgts$ Artifact | TgtPrompt$ Select target artifact | SpellDescription$ Destroy target artifact.
+SVar:DBChangeTargets:DB$ ChangeTargets | TargetType$ SpellAbility.singleTarget | ValidTgts$ Card,Emblem | TgtPrompt$ Select target spell or ability with a single target | SpellDescription$ Change the target of target spell or ability with a single target.
+SVar:DBCantBlock:DB$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | TargetMin$ 1 | TargetMax$ 2 | KW$ CARDNAME can't block. | IsCurse$ True | SpellDescription$ One or two target creatures can't block this turn.

```

Comment:
better code it as effect with static

Reply:
I looked around, and it's a mess, the effects that stop blocking seem to be all over the place and this is still overwhelmingly used for simple targeted "can't block" effects. Do we need to fix all of that now?

Reply:
Can also be fixed better later

Reply:
I'd prefer that, this seems like it would involve changing a sizeable number of cards -- and I would prefer to tackle that tomorrow at the soonest :)